.ddev/.setup/scripts/utils.sh

```
@@ -249,36 +551,180 @@ function create_symlinks_additional_extensions() {
+
+# Function to create the symlinks for a classic docroot.
+# It wires up the classic source symlink trio (typo3_src, typo3, index.php),
+# copies the shipped _.htaccess into the docroot, and symlinks the main
+# extension plus the fixture packages into typo3conf/ext/<key>.
+#
+# No Composer autoloader is used on purpose: the extension is linked like a TER
+# install so TYPO3 has to resolve its classes from the `autoload` key in
+# ext_emconf.php (v12/v13) resp. from the extension's composer.json (v14) —
+# which is exactly the fidelity this mode exists to test.
+function classic_create_symlinks() {
+    ( cd "$BASE_PATH/public"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)